src/main/groovy/org/jenkinsci/gradle/plugins/jpi/LicenseTask.groovy

```diff
@@ -66,8 +67,12 @@ class LicenseTask extends DefaultTask {
     }
 
     private Set<ResolvedArtifact> collectPomArtifacts() {
-        project.configurations
+        def detached = project.configurations
                 .detachedConfiguration(collectDependencies())
+        project.plugins.withId('org.jetbrains.kotlin.jvm') {
```

Comment:
I think you can **always** request the attribute and not add not do the "project.plugins.withId('org.jetbrains.kotlin.jvm')" check. It should work because "usage=java-runtime" is what we default to now anyway.

Reply:
Thanks! I noticed the tests passed if I didn't restrict by plugin, but wasn't sure if that would be preferred. I've made this change now.